kolibri/plugins/setup_wizard/assets/src/machines/wizardMachine.js

```diff
@@ -126,13 +143,16 @@ export const wizardMachine = createMachine({
     onMyOwnOrGroupSetup: {
       always: [
         {
+          // `cond` takes a function that returns a Boolean, continuing to the
+          // `target` when it returns truthy
           cond: isOnMyOwnOrGroup,
           target: 'defaultLanguage',
         },
         {
           cond: isGroupSetup,
           target: 'deviceName',
         },
+        // There is no fallback path here; if neither `cond` above is truthy, this will break
```

Comment:
This seems both correct, and also, potentially tricky. Are there other places where we are using a state machine that we have handled a similar scenario?

Reply:
Not that I know of -- ideally we would have some kind of error state for the Setup Wizard that we could put people into with a "start over" button and some information they can send us to help debug? This would require some design and new strings but could be a worthwhile thing to add? (cc @jtamiace)